### DISCUSSION TITLE
Crafting table recipes wont show up.

### DISCUSSION THREAD

**Discussion by glostone** [shedaniel/RoughlyEnoughItems#890] [Q & A]
When I view the recipes for an item, Crafting table wont show up. 
Auto crafters from other mods show up but not the vanilla crafting table.
![image](https://user-images.githubusercontent.com/85706815/171948027-dc689f17-aa02-457e-9225-3c88d1eb40a1.png)

**Comment by shedaniel** [comment]
Are you using a modpack? What mods do you have? Mind sharing your logs so we can look into this further?

**Comment by Tess-Dreams** [comment]
Hello. I hope it is alright to contribute to this conversation. I've had this exact same issue and upon a lot of disabling/enabling mods, the issue with glostone's and my modpack is some sort of incompatibility with the base Create mod. Roughly Enough Resources seem to work with it though, if that helps.

I'm also inexperienced with sharing logs on this site, so I will simply upload the .log files here. I hope that's okay.

[latest.log](https://github.com/shedaniel/RoughlyEnoughItems/files/8843165/latest.log)
[rei.log](https://github.com/shedaniel/RoughlyEnoughItems/files/8843171/rei.log)

(rei-issues.log was empty)

**Reply by Tess-Dreams** [reply]
Update to this: Deleting "Ingredient Extension API" fixes this issue. I assume it must've changed something inherent to recipes to break REI while a mod like Create was installed. REI + Ingredient Extension API (for CraftTweaker) on its own work well though.

**Comment by Shnupbups** [comment]
Couldn't manage to reproduce this issue with the modlist:
- Architectury API 4.4.68
- Cloth Config 6.2.62
- CraftTweaker 9.1.151
- Create 0.4.1+601
- Fabric API 0.55.1
- Ingredient Extension API 2.0.5
- Roughly Enough Items 8.2.471

Seems there's yet another layer to this odd interaction.

**Reply by jaredlll08** [reply]
Can you add farmers delight to that instance and see if anything changes please.

A quick look at the log shows that they are probably using the recipe jsons from the Forge counterpart where they have a custom ingredient type.

**Reply by Shnupbups** [reply]
Adding Farmer's Delight made no difference - still can't reproduce this issue.
